File: project-euler/0001-fizzbuzz.js
#!/usr/bin/env node

const range = (n) => Array(n).fill().map((_, i) => i)

const sum = (xs) => xs.reduce((x1, x2) => x1 + x2)

console.log(sum(range(1000).filter(n => n % 3 === 0 || n % 5 === 0)))

File: project-euler/0001-fizzbuzz.c
#include <stdio.h>
#include <stdlib.h>
int main()
{
    int i,a[999],sum=0;
    for(i=1;i<=999;++i)
    a[i-1]=i;
    for(i=0;i<999;++i)
    {
        if (a[i]%3==0 || a[i]%5==0)
        sum=sum+a[i];
    }
    printf("sum= %d",sum);
	  return 0;
}
